templates/cli/lib/commands/push.js.twig

```diff
@@ -1410,6 +1427,7 @@ push
     .command("collection")
     .alias("collections")
     .description("Push collections in the current project.")
+    .option(`-a, --attempts <numberOfAttempts>`, `Max number of attempts before timing out collection attribute operations, default: 30.`)
```

Comment:
attempts is definitely much better than debounce. The only thing is it's not just for attribute operations; it's for indexes too. So, maybe just:

Max number of attempts before timing out